test/Stackctl/AWS/CloudFormationSpec.hs

```diff
@@ -2,27 +2,95 @@ module Stackctl.AWS.CloudFormationSpec
   ( spec
   ) where
 
-import Stackctl.Prelude
+import Stackctl.Test.App
 
+import Amazonka.CloudFormation.DeleteChangeSet
+import Amazonka.CloudFormation.ListChangeSets
+import Amazonka.CloudFormation.Types.ChangeSetSummary
+import Blammo.Logging.Logger (LoggedMessage (..), getLoggedMessagesUnsafe)
+import qualified Data.Aeson.KeyMap as KeyMap
 import Data.List (isSuffixOf)
 import Stackctl.AWS.CloudFormation
-import Test.Hspec
 
 spec :: Spec
 spec = do
   describe "readParameter" $ do
-    it "refuses empty key" $ do
+    it "refuses empty key" $ example $ do
```

Comment:
til `example`, do you mean to leave it in here (as it seems to be here primarily for development purposes/better error messages, IIUC?). 

Don't care either way, just asking.

Reply:
I take it back, I'm guessing you meant to keep it, since you did add it to all of these tests that you didn't change. NVM

Reply:
Yeah, by moving to `Test.Hspec.Expectations.Lifted` so I can use them from within `TestAppT`, the body of any `it` becomes ambiguous. You can address it with an annotation anywhere, but `example` is meant for exactly this:

```hs
example :: IO a -> IO a
example = id
```